src/Symfony/Component/Routing/Tests/Loader/PhpFileLoaderTest.php

```diff
@@ -174,6 +174,9 @@ public function testRoutingConfigurator()
         $expectedCollection->add('buz', (new Route('/zub'))
             ->setDefaults(['_controller' => 'foo:act'])
         );
+        $expectedCollection->add('controller_class', (new Route('/controller'))
+            ->setDefaults(['_controller' => ['Acme\MyApp\MyController', 'myAction']])
+        );
```

Comment:
Thanks for adding this test :+1: 

Btw, is this syntax supported in the YAML too? We're using Symfony 2.8 and I look for ways to use better configs . I could not find it in Symfony 2.8 tests/documentaiton. Only strings are mentioned: https://symfony.com/doc/2.8/routing.html#controller-string-syntax

![image](https://user-images.githubusercontent.com/924196/184532537-c5acc135-9dff-46f3-8107-5e063e918671.png)


...

Just digging a bit deeper, it seems it was added in Symfony 4.4:
https://github.com/symfony/framework-bundle/commit/f0cc825f39c6bf028a14d16e9778daa314610786